src/macros.jl

```diff
@@ -60,8 +60,13 @@ macro polyvariable(args...)
   end
 
   # separate out keyword arguments
-  kwargs = filter(ex->isexpr(ex,:kw), extra)
-  extra = filter(ex->!isexpr(ex,:kw), extra)
+  if VERSION < v"0.6-"
```

Comment:
By the way, wouldn't it be easier to do `kwsymbol = VERSION < v"0.6.0-dev.1934" ? :kw : :(=) # changed by julia PR #19868` like done in the JuMP commit mentionned above ?